spa/src/components/account/SignIn/SignInForm.js

```diff
@@ -50,7 +49,7 @@ function SignInForm({ onSubmitSignIn, loading }) {
 
       <S.PasswordLabel hasError={errors.password}>
         Password
-        <a href={FORGOT_PASSWORD} data-testid="reset-password" tabindex="-1">
+        <a href={FORGOT_PASSWORD} data-testid="reset-password" tabIndex="-1">
```

Comment:
Why do we have this `tabIndex`? Shouldn't the user be able to access this via "tab"?

I think this link should be accessible with tabs
```suggestion
        <a href={FORGOT_PASSWORD} data-testid="reset-password">
```

Reply:
done